lib/features.json

```diff
@@ -318,5 +318,10 @@
     "name": "setSlotAsTransaction",
     "description": "use setslot as transaction instead of just hoping it'll work",
     "versions": ["1.17", "1.17.1"]
+  },
+  {
+    "name": "pingPacket",
```

Comment:
shouldn't be a thing, ping packet nisnt the same thing as transaction packet

Reply:
so should I just remove it or say that ping packet was added in 1.17?